app/components/github/install-link.js

```diff
@@ -11,6 +11,10 @@ export default Component.extend({
   organization: null,
   project: null,
 
+  installUrl: computed('githubAppUrl', function() {
+    return `${get(this, 'githubAppUrl')}/installations/new`;
+  }),
```

Comment:
#1481 deals with this directly, so maybe we wanna remove this when that is merged? I actually prefer the other approach. If we don't need the root app url for anything and only ever use this one, might as well set it directly.

Reply:
We do need the root app url because we also later want to provide a link for editing the existing configuration of an app installation directly.